docs/usage.rst

```diff
@@ -17,6 +17,7 @@ Current usage of ``pydocstringformatter``:
                                 [--capitalize-first-letter  --no-capitalize-first-letter]
                                 [--final-period  --no-final-period]
                                 [--quotes-type  --no-quotes-type]
+                                [--max-line-length MAX_LINE_LENGTH]
```

Comment:
It seems the CI fails here. You could try:
```python
cd docs
make html
```

That should update the docs.

Reply:
Did that. Unfortunately pipelines still complain